include/onnxruntime/core/framework/tensor_shape.h

```diff
@@ -14,6 +14,8 @@
 #pragma warning(push)
 // C4127: conditional expression is constant
 #pragma warning(disable : 4127)
+// C4324: structure was padded due to alignment specifier
+#pragma warning(disable : 4324)
```

Comment:
how about including inlined_containers.h here instead to avoid the warning pragmas?

Reply:
I did not want to create another dependency when it was not apparent